b2sdk/b2http.py

```diff
@@ -176,7 +179,15 @@ def add_callback(self, callback):
         """
         self.callbacks.append(callback)
 
-    def post_content_return_json(self, url, headers, data, try_count=5, post_params=None):
+    def post_content_return_json(
+        self,
+        url,
+        headers,
+        data,
+        try_count=20,
+        post_params=None,
+        force_timeout=None,
```

Comment:
that's a bit awkward, I think this parameter should be named `timeout` and have it's default value computed inside of the function just like it is now.

Anyway, typing would improve the current nomenclature. `force_timeout`, without even being mentioned in the docstring should be typed, someone could use a `bool` as its value as it is now.

Reply:
_timeout + type hint